ush/soca/marine_recenter.py

```diff
@@ -263,7 +263,28 @@ def finalize(self):
             mem_dir_list.append(mem_dir_real)
             copy_list.append([f'ocn.recenter.incr.{str(mem)}.nc',
                               os.path.join(mem_dir_real, incr_file)])
-
+        # Copy the ensemble variance
+        ensvar_date = self.task_config.MARINE_WINDOW_END.strftime('%Y-%m-%dT%H:%M:%SZ')
+        stats_dir = os.path.join(self.task_config.ROTDIR,
+                                 f'enkf{RUN}.{PDYstr}',
+                                 f'{cyc}',
+                                 'ensstat',
+                                 'analysis',
+                                 'ocean')
```

Comment:
Is there a template to make this path from? If not, we'll need to add it

Reply:
They are used to create the paths under COM, and start under https://github.com/NOAA-EMC/global-workflow/blob/940e3539c174b05564f0652aad4bb41af0e8f697/parm/config/gfs/config.com#L84
The actual paths are declared in the job script and then accessed from the task_config in the script. We don't have to hold things up for that (it requires a PR to g-w) but it's something to address in an issue. I can do it.

Reply:
Unless we want to use` analysis/ocean`?

Reply:
Oh, cool, I didn't know this, thank you for pointing to it!
It _may be_ COM_OCEAN_ANALYSIS_TMPL when MEMDIR is "ensstat"? I'm not sure.
Thank you for offering to do it! 🙏

Reply:
That is, `COMOUT_OCEAN_ANALYSIS` and `COMOUT_ICE_ANALYSIS`, which are already available?

Reply:
I think @AndrewEichmann-NOAA might be talking about making use of this:
https://github.com/NOAA-EMC/global-workflow/blob/940e3539c174b05564f0652aad4bb41af0e8f697/jobs/JGLOBAL_MARINE_ANALYSIS_LETKF#L31

Is that correct @AndrewEichmann-NOAA ?

Reply:
@guillaumevernieres More like https://github.com/NOAA-EMC/global-workflow/blob/940e3539c174b05564f0652aad4bb41af0e8f697/jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN#L29

Reply:
I saw that the atmospheric ensemble stats go to a similar directory as in this PR:
`enkfgdas*/[hour]/ensstat/[ocean|atm|ice]/analysis`
while the ensemble members go to
`enkfgdas*/[hour]/mem*/[ocean|atm|ice]/analysis`

Note that this is different from `gdas*/[hour]/[ocean|ice]/analysis/` which I think correspond to `COMOUT_OCEAN_ANALYSIS` and `COMOUT_ICE_ANALYSIS`?

Reply:
Ah, right. I'd have to dig into how enkfgdas is handled.

Reply:
I'm happy to approve this and handle the templating later.

Reply:
Just wrote the above but see that @JessicaMeixner-NOAA flagged this as a v17 issue ... Oops. I'll head scratch a bit more before merging this work.

Reply:
You flagged it correctly @JessicaMeixner-NOAA .

Reply:
yep, would be good to still have spread diagnostics like the ones we looked at this week when we don't run letkf (in retro and parallel runs and in operations eventually).